addons/html_editor/static/src/main/toolbar/toolbar_plugin.js

```diff
@@ -151,7 +151,7 @@ export class ToolbarPlugin extends Plugin {
             groupId: "expand_toolbar",
             commandId: "expandToolbar",
             description: _t("Expand toolbar"),
-            icon: "fa-ellipsis-v",
+            icon: "oi oi-ellipsis-v",
```

Comment:
Same here.
Also, you should replace `fa-fw` with `oi-fw` when this new icon uses it :)
![Capture d’écran 2025-04-07 à 11 03 14](https://github.com/user-attachments/assets/59b3a814-7ad7-4f00-90d6-d9973fcb3b31)